src/applications/personalization/dashboard/utils/claims-helpers.js

```diff
@@ -236,6 +236,26 @@ export function isClaimComplete(claim) {
   return claim.attributes.decisionLetterSent || claim.attributes.phase === 8;
 }
 
+export function isLighthouseClaimComplete(claim) {
+  return (
+    claim.attributes.decisionLetterSent ||
```

Comment:
If you are attempting to determine if the claim is open or closed, I wouldn't use this flag as it can be `true` while the claim is not in a 'Closed' state. If you are looking to cover all bases, then it's best to check that the `closeDate` is set to `null` instead

Reply:
So I was actually looking at the version of this function that is in the CST and I'm realizing why we are checking for decisionLetterSent. For the claim cards we have a concept of "In Progress", that exists separate from the status of the claim. If a claim is "In Progress", then there are certain icons that can show in the card to indicate the specific state the claim is in. If the claim is not "In Progress" then we don't show most of these. With that being said, you should leave the code the way it is